python/requirements-wheel-test.txt

```diff
@@ -0,0 +1,12 @@
+cffi
+cython
+hypothesis==5.0; python_version <= "3.5.2"
```

Comment:
Why "3.5.2" exactly? Did something happen in 3.5.3?

Reply:
Yes, the python typing incompatibilities happened at that version.